content/docs/services/cloud-gov-service-account.md

```diff
@@ -0,0 +1,35 @@
+---
+menu:
+  docs:
+    parent: services
+title: cloud.gov service account
+name: "cloud-gov-service-account"
+description: "Manage cloud.gov service accounts with access to your organization"
+status: "Beta"
+aliases:
+- /docs/services/deployer-account/
+---
+
+To set up your application to be deployed with an automated system, you need a deployer account that has access to push your application.
```

Comment:
"need" is a little strong. I'd say something like "you should use an account that has access to push your application that's not associated with a specific user (to insulate against people leaving your organization, changing projects, etc.)"